core/components/css-utilities/Overflow/Overflow.story.tsx

```diff
@@ -12,62 +12,318 @@ export const overflow = () => {
+                <Text className="mt-3">Horizontal scrollbar always present</Text>
+              </div>
+            </div>
+          </div>
+        </Card>
+
+        {/* Vertical Overflow Examples */}
+        <Card className="mb-8 p-6">
+          <Heading size="s">Vertical Overflow (Y-axis)</Heading>
+          <Text className="mt-3 mb-5">These utilities control vertical overflow only.</Text>
+
+          <div className="d-flex flex-wrap">
+            {/* overflow-y-auto */}
+            <div className="mr-8 mb-6 w-25">
+              <Text weight="strong" className="mb-3">
+                overflow-y-auto
+              </Text>
+              <Text appearance="subtle">(Vertical scrollbar appears as needed)</Text>
+              <div className="position-relative">
+                <div className="overflow-y-auto p-4 bg-secondary-lightest border rounded-05 vh-25 mb-5">{longText}</div>
+              </div>
+            </div>
+
+            {/* overflow-y-hidden */}
+            <div className="mr-8 mb-6 w-25">
+              <Text weight="strong" className="mb-3">
+                overflow-y-hidden
+              </Text>
+              <Text appearance="subtle">(Vertical content is clipped)</Text>
+              <div className="position-relative">
+                <div className="overflow-y-hidden p-4 bg-secondary-lightest border rounded-05 vh-25 mb-5">
+                  {longText}
+                </div>
+              </div>
+            </div>
+
+            {/* overflow-y-visible */}
+            <div className="mr-8 mb-6 w-25">
+              <Text weight="strong" className="mb-3">
+                overflow-y-visible
+              </Text>
+              <Text appearance="subtle">(Vertical content flows outside)</Text>
+              <div className="position-relative overflow-hidden">
+                <div className="overflow-y-visible p-4 bg-secondary-lightest border rounded-05 vh-25 mb-5">
+                  {longText}
+                </div>
+              </div>
+            </div>
+
+            {/* overflow-y-scroll */}
+            <div className="mb-6 w-25">
+              <Text weight="strong" className="mb-3">
+                overflow-y-scroll
+              </Text>
+              <Text appearance="subtle">(Vertical scrollbar always present)</Text>
+              <div className="position-relative">
+                <div className="overflow-y-scroll p-4 bg-secondary-lightest border rounded-05 vh-25 mb-5">
+                  {longText}
+                </div>
+              </div>
+            </div>
+          </div>
+        </Card>
+
+        {/* Practical Example */}
+        <Card className="p-6">
+          <Heading size="s">Practical Example</Heading>
+          <Text className="mt-3 mb-5">A real-world example of using overflow utilities in a UI component:</Text>
+
+          <div className="d-flex flex-wrap">
+            <div className="mr-8 mb-6 w-25">
+              <Text weight="strong" className="mb-3">
```

Comment:
Please add some spacing in this example 
<img width="625" alt="image" src="https://github.com/user-attachments/assets/a8713ac5-de7c-43d0-a659-38a851e8c7d2" />